src/api/dataPlanes.ts

```diff
@@ -10,29 +12,52 @@ export interface BaseDataPlaneQuery {
     data_plane_name: string;
     id: string;
     reactor_address: string;
+    cidr_blocks: string[] | null;
 }
 
+const COLUMNS = ['data_plane_name', 'id', 'reactor_address', 'cidr_blocks'];
+
+const QUERY = COLUMNS.join(',');
+
 const getDataPlaneOptions = async () => {
     const data = await supabaseRetry(
         () =>
             supabaseClient
                 .from(TABLES.DATA_PLANES)
-                .select('data_plane_name,id,reactor_address')
+                .select(QUERY)
                 .order('data_plane_name'),
         'getDataPlaneOptions'
     ).then(handleSuccess<BaseDataPlaneQuery[]>, handleFailure);
 
     return data;
 };
 
+const getDataPlanesForTable = (
+    catalogPrefix: string,
+    pagination: any,
+    searchQuery: any,
+    sorting: SortingProps<any>[]
+) => {
+    return defaultTableFilter<BaseDataPlaneQuery>(
+        supabaseClient
+            .from(TABLES.DATA_PLANES)
+            .select(QUERY)
+            .ilike('data_plane_name', `ops/dp/private/${catalogPrefix}%`),
```

Comment:
The `DATA_PLANE_PREFIX` constant should be used in place of `'ops/dp/'`. That said, should a constant be defined for the private data-plane prefix at this time? It would be the analog to `PUBLIC_DATA_PLANE_PREFIX`.